.github/workflows/ci.yml

```diff
@@ -27,6 +27,24 @@ jobs:
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install -q --no-cache-dir -e .[complete]
         python -m pip list
+    - name: Unit Tests
+      run: |
+        python -m pip install pytest
```

Comment:
could we add this to a `[test]` extra? We do something similar in pyhf

https://github.com/scikit-hep/pyhf/blob/master/setup.py#L25

Reply:
I would strongly recommend doing exactly this (in a separate PR at these should be atomic).

Reply:
This is done in PR #32